### PR TITLE
Iar 60/create database and connect with backend

### PR DESCRIPTION
**Pull request by TomasBenes**
Connection with database is set up

### CODE REVIEW

**Review by JirinaGFA — DISMISSED** [review]
great, works!

File: src/entities/User.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  @Column()
+  username: string;
+
+  @Column()
+  email: string;
+
+  @Column()
+  password: string;
+}
```
Comment (JirinaGFA) [inline src/entities/User.ts]: date_created is missing

**Review by JirinaGFA — CHANGES_REQUESTED** [review]
date_created in dtb is missing

**Review by SimProch — APPROVED** [review]
Make sure to incorporate Jirinka's requested changes

**Review by JirinaGFA — APPROVED** [review]

**Pull request merged**